xdsl/dialects/emitc.py

```diff
@@ -180,8 +191,8 @@ def get_num_dims(self) -> int:
     def get_shape(self) -> tuple[int, ...]:
         return tuple(i.data for i in self.shape.data)
 
-    def get_element_type(self) -> EmitCArrayElementType:
-        return self.element_type
+    def get_element_type(self) -> EmitCArrayElementTypeCovT:
+        return cast(EmitCArrayElementTypeCovT, self.element_type)
```

Comment:
Let's make `self.element_type: EmitCArrayElementTypeCovT`?

Reply:
yup :) done in a6142bb